api/Animation.json

```
@@ -1339,7 +1339,7 @@
       },
       "remove_filling_animation": {
         "__compat": {
-          "description": "Browsers automatically remove <a href='https://developer.mozilla.org/docs/Web/API/Animation#Automatically_removing_filling_animations'>indefinite filling animations</a>.",
+          "description": "Browsers automatically remove indefinite filling animations.",
```

While we're here, we should drop the punctuation

```suggestion
          "description": "Browsers automatically remove indefinite filling animations",
```